python/manylinux1/build_arrow.sh

```diff
@@ -75,6 +75,9 @@ for PYTHON_TUPLE in ${PYTHON_VERSIONS}; do
 
     echo "=== (${PYTHON}) Test the existence of optional modules ==="
     $PIP install -r requirements.txt
+    if [ "$( cut -d ',' -f 1 <<< "$PYTHON_TUPLE" )"  == "2.7" ]; then
```

Comment:
Please add this directly to the requirements file using environment markers (`futures; python_version < 3`)

Reply:
Then whit [one](https://github.com/apache/arrow/blob/master/ci/travis_script_python.sh#L97) should be removed too.